R/GetReport.R

```diff
@@ -26,13 +27,14 @@
 #' @export
 #'
 
-GetReport <- function(report.id,interval.seconds=10,max.attempts=3,print.attempts=TRUE) {
+GetReport <- function(report.id,interval.seconds=10,max.attempts=3,print.attempts=TRUE,format="json") {
```

Comment:
Same comment here....since GetReport is a public function, I'm not sure that `format` should be public. That said, I'm not sure it shouldn't be public either!

Since `GetReport` is nested the context of `QueueDatawarehouse`, couldn't/shouldn't we just pass `format` from the `QueueDataWarehouse` scope into the `ApiRequest` call?